docs/specs/backend_internal.yaml

```diff
@@ -1,586 +1,1266 @@
-openapi: 3.0.1
+openapi: 3.1.0
 info:
   title: 3scale Backend Internal API
   version: 1.0.0
 servers:
-- url: http://localhost:3001/internal
+  - url: http://localhost:3001/internal
 paths:
   /services/{service_id}/alert_limits/:
     get:
       tags:
-      - Alert limits
+        - Alert limits
```

Comment:
any important reason to change the indentation? :zipper_mouth_face:

Reply:
No... just my IDE does that automatically, apparently. Or, maybe it's https://editor-next.swagger.io/ that I'm using.

I'll try to revert those before opening the PR for review.

Reply:
May the change is not needed but it's fine to leave it IMO.

Reply:
This is two bytes of waste, and loss of priceless screen estate, unacceptable! :angel:

Reply:
Your point is irrefutable. However, in order to keep consistency with the rest of the file, those two spaces should be removed also from all new sections added in this PR.

Lacking of consistence inside within the same file... how unacceptable, right?

Reply:
If such thing happens, we'll have to remove all copies of the source code of this project and then start it over. We must be careful lest nobody remains to tell the story.

Reply:
If you value consistency, yes, definitely we should.